app/containers/Achievements/UniqueAchievement.jsx

```diff
@@ -74,62 +76,66 @@ const UniqueAchievement = ({
   const onMouseLeave = useCallback(() => changeVisibility(false), []);
 
   return (
-    <Bage>
-      <ImageBlock>
-        {reached && (
-          <div
-            onMouseEnter={onMouseEnter}
-            onMouseLeave={onMouseLeave}
-            className="position-relative"
-          >
-            {visible && (
-              <NFTInformation
-                id={id}
-                locale={locale}
-                contractAddress={contractAddress}
-                ipfsHash={achievementURI}
-              />
+    <>
+      {(reached || achievementsType == LIMITED_EDITION_NFT_TYPE) && (
```

Comment:
We added a new achievement type that we mint manually for campaigns. We don't need to show anything for those NFTs if the user does not have them.